Rearrange Readme

avoidr opened the pull request:
This is basically how I would like to see the readme.

Do we want to make the operating systems bold again? I wasn't sure.
Gonna `s/peer2peer/peer-to-peer/`, too, I think.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/utox/utox/1479)
<!-- Reviewable:end -->

avoidr commented:
Oh, and also how to recommend aTox in operating systems. I feel like it should be done.

redmanmale commented on README.md:
```diff
@@ -2,59 +2,51 @@
+[**Installing**](docs/INSTALL.md) **|** [**Compiling**](docs/BUILD.md) **|** [**Screenshots**](screenshots/INDEX.md) **|** [**Changelog**](CHANGELOG.md) **|** [**TokTok Site**](http://toktok.github.io/) **|** [**Toxcore Spec**](https://toktok.github.io/spec)
 
-You shouldn't have to choose a new client for every device: So µTox works everywhere! On every major system **Windows**, **macOS**, **Linux**, **Android**, **FreeBSD**, **NetBSD**, **OpenBSD**, and even **DragonFlyBSD**.
+Tox is a peer2peer, end-to-end encrypted, instant messaging protocol, with end-to-end encryption as its default and *only* mode.
```
I think it should be either peer-to-peer or end2end, not two different ways like now.

redmanmale commented on README.md:
```diff
@@ -2,59 +2,51 @@
-- Emoticons
 
 Just like Toxcore, µTox is still alpha software, so you may encounter bugs, or maybe a crash or two. µTox also needs your help, if you do encounter any bugs or problems please [open an issue](https://github.com/uTox/uTox/issues/new).
+If you do not have a GitHub account, you may also send an email directly to avoidr.
```
Add your email here or link to contacts at the bottom?

redmanmale commented on screenshots/INDEX.md:
```diff
@@ -0,0 +1,13 @@
+# Screenshots
```
Maybe we should update our screenshots.

redmanmale commented on README.md:
```diff
@@ -2,59 +2,51 @@
 The lightweight [Tox](https://github.com/TokTok/toxcore) client.
 
-[**TokTok Site**](http://toktok.github.io/) **|** [**Installing**](docs/INSTALL.md) **|** [**Changelog**](CHANGELOG.md) **|** [**Compiling**](docs/BUILD.md) **|** [**Toxcore Spec**](https://toktok.github.io/spec)
+[**Installing**](docs/INSTALL.md) **|** [**Compiling**](docs/BUILD.md) **|** [**Screenshots**](screenshots/INDEX.md) **|** [**Changelog**](CHANGELOG.md) **|** [**TokTok Site**](http://toktok.github.io/) **|** [**Toxcore Spec**](https://toktok.github.io/spec)
```
Right now `Changelog` leads to the big fat **deprecated** page.
Maybe we should link [release note](https://github.com/avoidr/uTox/tree/readme/release_notes) here.

avoidr commented:
> -You shouldn't have to choose a new client for every device: So µTox works everywhere! On every major system **Windows**, **macOS**, **Linux**, **Android**, **FreeBSD**, **NetBSD**, **OpenBSD**, and even **DragonFlyBSD**.
> +Tox is a peer2peer, end-to-end encrypted, instant messaging protocol, with end-to-end encryption as its default and *only* mode.
> 
> I think it should be either peer-to-peer or end2end, not two different ways like now.

Yes, I was thinking the same. I'll make it peer-to-peer.

>  Just like Toxcore, µTox is still alpha software, so you may encounter bugs, or maybe a crash or two. µTox also needs your help, if you do encounter any bugs or problems please [open an issue](https://github.com/uTox/uTox/issues/new).
> +If you do not have a GitHub account, you may also send an email directly to avoidr.
> 
> Add your email here or link to contacts at the bottom?

Good idea! I'll link to contacts.

> Maybe we should update our screenshots.

Maybe, though I don't like binary data in code repos, actually. Especially not huge (the win10 screenshot).

avoidr commented:
> -[**TokTok Site**](http://toktok.github.io/) **|** [**Installing**](docs/INSTALL.md) **|** [**Changelog**](CHANGELOG.md) **|** [**Compiling**](docs/BUILD.md) **|** [**Toxcore Spec**](https://toktok.github.io/spec)
> +[**Installing**](docs/INSTALL.md) **|** [**Compiling**](docs/BUILD.md) **|** [**Screenshots**](screenshots/INDEX.md) **|** [**Changelog**](CHANGELOG.md) **|** [**TokTok Site**](http://toktok.github.io/) **|** [**Toxcore Spec**](https://toktok.github.io/spec)
> 
> Right now `Changelog` leads to the big fat **deprecated** page.
> Maybe we should link [release note](https://github.com/avoidr/uTox/tree/readme/release_notes) here.

Yeah, I wasn't sure when to make that change. But I think we're good now.

Thanks for your review!

redmanmale commented on README.md:
```diff
@@ -2,59 +2,51 @@
-  - Audio
+- 1-on-1 chats
   - Text
     - Chat History
```
I'd rather move it to the same level as others: third list level is overkill.

avoidr commented:
- s/peer2peer/peer-to-peer/
- added link to contacts
- replaced changelog with release notes

Added a `release_notes/INDEX.md`, not sure if I like it that way, probably yes, if we're going to use md, anyway. So it all looks the same.

But notice that github broke their markdown presentation, so maybe we don't want to rely on that and use simple text, or whatever. I wanna wait and see if they fix their shit. @unames don't link to profiles, commit IDs aren't linked anymore, and newlines aren't respected (now newlines behave like in LaTeX). So now we have dumbass plaintext, except without it actually being dumb—just ass.

avoidr commented:
> I'd rather move it to the same level as others: third list level is overkill.

Ok, like this?
